pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc

```diff
@@ -17,15 +17,15 @@ The configuration file enables you to import content from a Git repository into
 The {ProductShortName} configuration file defines the following parameters:
 
 * `server` The URL of the {ProductShortName} server.
-* `repository`
+* `repository` The name of the repository. It is recommended not to include spaces in the repository name.
 * `variants` A multi-valued structure with the following child members:
 +
-** `name` (optional) A unique ID that displays in URLs on the {ProductShortName} site when you view modules and assemblies.
+** `name` (optional) A unique ID that displays in URLs on the {ProductShortName} site when you view modules and assemblies. If no value is provided for `name`, this field is defined as *DEFAULT*.
 +
 For example, if you specify `name: rhel_docs`, and one of your files is named [filename]`foo.adoc`, the module's URL in {ProductShortName} is:
-`\http://<_{ProductShortName}__url>/content/repositories/**rhel_docs**/entities/modules/getting_started/foo.adoc.preview`
+`\http://<_{ProductShortName}__url>/content/repositories/*repository*/entities/modules/getting_started/foo.adoc.preview`
```

Comment:
@emarcusRH I don't understand why you changed `rhel_docs` to `repository` here. Isn't this supposed to be an example of what is written in line 23?

Reply:
Lisa Davidson requested/corrected this to make it generic